RELEASES.md

```diff
@@ -18,6 +18,7 @@ Version 0.9.1 (2023-02-23)
 * Toyota C-HR 2021 support thanks to eFiniLan!
 * Toyota C-HR Hybrid 2022 support thanks to Korben00!
 * Volkswagen Crafter and MAN TGE 2017-23 support thanks to jyoung8607!
+* Added Honda HR-Vezel non US
```

Comment:
You don't need to call it non-US, especially since your car is North American. I checked and it says the ZR-V is released as the 2023 HR-V: https://en.wikipedia.org/wiki/Honda_ZR-V

Reply:
The reason it is stated as such is because it references the non us 2023 hr-v (the us model is different from the rest of the world).
I think for the code, the 2023 us hrv should be listed as a 2023 zrv, where we specify somewhere the naming convention...